gen/main.go

```diff
@@ -29,12 +29,14 @@ func main() {
 		types.MsgMeta{},
 		types.ActorV4{},
 		types.ActorV5{},
-		types.MessageReceipt{},
+		// types.MessageReceipt{}, // Custom serde to deal with versioning.
```

Comment:
Just remove this entirely?

Reply:
If you really want to. I think it's a useful strategic marker, as this case is exceptional due to versioning.

Reply:
I'll keep it. If you feel strongly about it, let me know.